src/components/Community/AddPost/PostMatchReview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+interface MatchReviewProps {
```

Comment:
컴포넌트 이름과 동일하게 가려면 이렇게 되어야겠네요!
```suggestion
interface PostMatchReviewProps {
```